src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -266,7 +267,19 @@ protected override void OnScriptFileChanged(object sender, FileSystemEventArgs e
                     {
                         TraceLevel level = (TraceLevel)logData["lvl"];
                         var evt = new TraceEvent(level, message);
-                        traceWriter.Trace(evt);
+
+                        // Node captures the AsyncLocal value of the first invocation, which means that logs
+                        // are correlated incorrectly. Here we'll overwrite that value with the correct value
```

Comment:
You're saying we have a log correlation issue currently? Not sure I understand this.